bin/ch-fromhost

```diff
@@ -1,9 +1,19 @@
 #!/bin/sh
-
+#
+# There are three primary uses of ch-fromhost:
+#
+#   1) inject arbitrary files from the host into the container;
+#
+#   2) inject host-built, loadable libfabric (OFI) dynamic shared object (dso)
+#      providers into an image with a MPI built against libfabric to use; and
+#
+#   3) inject nvidia libraries necessary to leverage GPUs with
+#      nvidiia-container-cli.
+#
 # The basic algorithm here is that we build up a list of file
 # source:destination pairs separated by newlines, then walk through them and
-# copy them into the image. We also maintain a list of directories to create
-# and a list of file globs to remove.
+# copy them into the image. We also maintain a list of directories and ldconfig
+# conf files to create, and a list of file globs to remove.
```

Comment:
Does `ldconfig` need to be told about the new config files? Maybe you do this below ...

Reply:
Ah, the assumption is that the container's `/etc/ld.so.conf` has`/etc/ld.so.conf.d/*`, which would pick up the file `ch-ofi.conf` we create.

Perhaps we shouldn't assume and explicitly check for and add it?

Reply:
Yes, probably safest not to assume that line is present and/or `/etc/ld.so.conf` exists.